tests/openthread_ncp/Makefile

```diff
@@ -0,0 +1,58 @@
+APPLICATION = openthread_ncp
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= samr21-xpro
+
+# These are the board that OpenThread stack has been tested on
+BOARD_WHITELIST := samr21-xpro iotlab-m3 fox iotlab-a8-m3
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# Comment this out to disable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+CFLAGS += -DDEVELHELP -Wall
+
+# Change this to 0 show compiler invocation lines by default:
+QUIET ?= 1
+
+# NCP: A Network Co-Processor is used with wpantund software. wpantund is a user-space network
+# interface driver/daemon that provides a native IPv6 network interface to a low-power wireless Network Co-Processor
+# FTD (see openthread_test_ftd): NCP needs Full Thread Device to act as a router in the network
+USEPKG += openthread
+USEMODULE += openthread_contrib
+USEMODULE += libmbedcrypto
+USEMODULE += libopenthread-ftd
+USEMODULE += libopenthread-ncp-ftd
```

Comment:
I use libopenthread-ftd because this is the name of .a library:
libopenthread-ftd.a
libopenthread-ncp-ftd.a
So it makes sense for me to keep these name

Yes contrib et crypto could be dependencies of PKG openthread, where should I add PKG dependencies?
Makefile.dep
Something like:
```
ifneq (,$(filter openthread,$(USEPKG)))
USEMODULE += openthread_contrib
USEMODULE += libmbedcrypto
endif
```